backend/views/syson-diagram-general-view/src/main/java/org/eclipse/syson/diagram/general/view/GeneralViewDescriptionProvider.java

```diff
@@ -40,49 +26,19 @@
  * @author arichard
  */
 @Configuration
-public class GeneralViewDescriptionProvider implements IEditingContextRepresentationDescriptionProvider {
-
-    private static final String GENERAL_VIEW_DIAGRAM_ID = "GeneralViewDiagram";
-
-    private final IViewConverter viewConverter;
-
-    private final EPackage.Registry ePackagesRegistry;
-
-    private final IInMemoryViewRegistry inMemoryViewRegistry;
+public class GeneralViewDescriptionProvider extends AbstractViewDescriptionProvider implements IEditingContextRepresentationDescriptionProvider {
```

Comment:
Could you remove `implements IEditingContextRepresentationDescriptionProvider` please as it is already handled by `AbstractViewDescriptionProvider`?
Could you also remove from every other class inheriting `AbstractViewDescriptionProvider`.

Reply:
I thought that Spring boot required explicitly this implements directive to inject properly the list of `IEditingContextRepresentationDescriptionProvider` in `EditingContextSearchService`
I will remove this.

Reply:
Done